container-search/src/main/java/com/yahoo/search/query/Select.java

```diff
@@ -84,7 +87,7 @@ public String getWhereString(){
 
 
     /** Set the grouping-string for the query. Must be a JSON-string, with the format described in the Select Reference doc - https://docs.vespa.ai/documentation/reference/select-reference.html. */
```

Comment:
Could you add html markup here so the url is clickable?
Also, add line breaks.

Reply:
👍